pkg/client/saramaclient.go

```
@@ -49,6 +56,48 @@ func (s *SaramaClient) ListBrokers() map[int]string {
 	return brokerMap
 }
 
+func (s *SaramaClient) ListConsumerGroups() (map[string]string, error) {
+	return s.admin.ListConsumerGroups()
+}
+
+func (s *SaramaClient) GetConsumerGroupsForTopic(groups []string, topic string) (chan string, error) {
+	var wg sync.WaitGroup
+	consumerGroupsChannel := make(chan string, len(groups))
+
+	if len(topic) == 0 {
+		for i := 0; i < len(groups); i++ {
+			consumerGroupsChannel <- groups[i]
+		}
+		close(consumerGroupsChannel)
+
+		return consumerGroupsChannel, nil
+	}
+
+	for i := 0; i < len(groups); i++ {
+		wg.Add(1)
+		go func(i int, wg *sync.WaitGroup) {
+			defer wg.Done()
+			groupDescription, _ := s.admin.DescribeConsumerGroups([]string{groups[i]})
```

let's not ignore errors